お部屋の内見一覧ページにスコア入力数を表示する

uchihiro04 opened the pull request:
## 概要 
お部屋の内見一覧ページに、部屋ごとのスコア入力数を表示しました。

## スクリーンショット
* スコア入力数の表示（赤枠で強調）
<img width="700" alt="230711_スコア入力数" src="https://github.com/uchihiro04/house-viewing-score-log/assets/77523896/e55b5b82-eb88-4d58-8f43-572a9bd5d9c2">

デベロッパツールで「iPhone SE」の画面（375 × 667）を想定して表示した場合
<img width="200" alt="230711_スコア入力数（スマートフォン）" src="https://github.com/uchihiro04/house-viewing-score-log/assets/77523896/a64639b3-96ef-47ae-8d7c-22e7c51c1dce">

## 関連Issue
- #79 

Close #79

uchihiro04 committed:
#79 スコア入力数を表示

お部屋の内見一覧ページについて、スコア入力数を表示しました。

Yuma-Nagaoka commented on app/views/house_viewings/rooms/index.html.slim:
```diff
@@ -3,8 +3,8 @@ table
-        td
-          = link_to room.name, new_house_viewing_room_score_path(room_id: room.id)
+        td = link_to room.name, new_house_viewing_room_score_path(room_id: room.id)
+        td = "スコア入力数: #{room.number_of_scores}"
```
ここ以前メソッド化しないという風にしたので、`room.scores.length`にしましょう。
でないと他の`scores.length`も`number_of_scores`に置換しないといけなくなるので

uchihiro04 replied:
ご指摘いただいた通り、以前似たような状況になった際は`scores.length`をメソッド化しない方向にしていました……。

メソッド化をやめ、`room.scores.length`をそのまま書く形に変更しました。該当するコミットは[4e202bd](https://github.com/uchihiro04/house-viewing-score-log/pull/133/commits/4e202bdc55746214cfbe9e2dd721874a0502be2a)です。

Yuma-Nagaoka commented on app/controllers/house_viewings/rooms_controller.rb:
```diff
@@ -4,7 +4,7 @@ module HouseViewings
     def index
       @house_viewing = HouseViewing.find_by!(uuid: params[:house_viewing_uuid])
-      @rooms = @house_viewing.rooms.order(:created_at)
+      @rooms = @house_viewing.rooms.includes([:scores]).order(:created_at)
```
includes(:scores)でいけないですか？

uchihiro04 replied:
ご指摘いただいた通り、`includes(:scores)`で問題なく動きました。gem（bullet）に指摘を受けた際に提案された`includes([:scores])`をそのまま使った形です。鵜呑みにしないように気をつけたいと思います。

いただいたレビューをもとに修正を行いました。該当するコミットは[9031ff9](https://github.com/uchihiro04/house-viewing-score-log/pull/133/commits/9031ff9cda31fa4ba3248ffe74ab71252ddcf281)です。

uchihiro04 committed:
#79 メソッドを削除

number_of_scoresメソッドについて、以前同じような状況の際にメソッド化をしなかったため、今回も以前の方針に従いメソッド化をやめコードを直接記載する形にしました。

uchihiro04 committed:
#79 不要な大括弧を削除
